analyzers/src/SonarAnalyzer.VisualBasic/SymbolicExecution/Roslyn/PublicMethodArgumentsShouldBeCheckedForNull.cs

```diff
@@ -85,7 +85,7 @@ public override void Visit(SyntaxNode node)
         public override void VisitIdentifierName(IdentifierNameSyntax node) =>
             DereferencesMethodArguments |=
                 argumentNames.Contains(node.GetName())
-                && node.HasAncestorOfKind(AwaitExpression,
+                && node.HasAncestor(AwaitExpression,
```

Comment:
[T0028](https://github.com/SonarSource/sonar-dotnet/issues/9161)
```suggestion
                && node.HasAncestor(
                    AwaitExpression,
```

Reply:
You can fix this in your next PR, I'll merge this one now as I need to nuke the whole helper file